src/EverCrypt_AEAD.c

```diff
@@ -26,8 +26,15 @@
 #include "EverCrypt_AEAD.h"
 
 #include "internal/Vale.h"
+#ifdef HACL_CAN_COMPILE_AESNI_PCLMUL
```

Comment:
I feel like the majority of the changes in this file are hand-written... is this intentional?

Reply:
No, there's no reason to hand-write the changes other than not noticing the F* source file.